docs/reference/aggregations/metrics/median-absolute-deviation-aggregation.asciidoc

```diff
@@ -0,0 +1,185 @@
+[[search-aggregations-metrics-median-absolute-deviation-aggregation]]
+=== Median Absolute Deviation Aggregation
+
+This `single-value` aggregation approximates the https://en.wikipedia.org/wiki/Median_absolute_deviation[median absolute deviation]
+of its search results.
+
+Median absolute deviation is a measure of variability that is useful for
+describing data that may or may not be normally distributed. For data that is
+not normally distributed, it can be more descriptive than standard deviation. It
+is calculated as the median of each data point's deviation from the sample
+median. That is, for a random variable X, the median absolute deviation is
+median(|median(X) - X~i~|).
+
```

Comment:
Perhaps a sentence (or addition to existing sentence) somewhere that describes MAD as being robust/resistant to change by outliers?

Reply:
👍